addon-resizer/nanny/estimator.go

```diff
@@ -56,3 +56,18 @@ func (e LinearEstimator) scaleWithNodes(numNodes uint64) *api.ResourceRequiremen
 		Requests: requests,
 	}
 }
+
+// LogarithmicEstimator estimates the amount of resources in the way that
+// prevents from frequent updates but may end up with larger resource usage
+// than actually needed (though no more than 50%).
+type LogarithmicEstimator struct {
+	linearEst LinearEstimator
+}
+
+func (e LogarithmicEstimator) scaleWithNodes(numNodes uint64) *api.ResourceRequirements {
+	n := uint64(16)
+	for n <= numNodes {
+		n = n * 3 / 2
+	}
+	return e.linearEst.scaleWithNodes(n)
```

Comment:
I can do a copy/paste if you think it would be more readable;)

Regarding speter.net/go/exp/math/dec/inf the general problem with this project is that there is no godeps, so it's not possible to compile it:(